libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -847,6 +870,29 @@ void AP_Camera::convert_params()
         // CAM_TRIGG_TYPE was set to Relay, GoPro or Mount
         cam1_type = cam_trigg_type + 1;
     }
+#if AP_CAMERA_RUNCAM_ENABLED
+    // find vehicle's top level key
```

Comment:
Need comments in here with the `PARAMETER_CONVERSION` tag and relevant dates

Reply:
Fixed